static/assets/Rendering/CubismRenderer.ts

```diff
@@ -241,7 +241,7 @@ export default class CubismRenderer extends Component {
   //#endregion
 
   //#region Meshes
-  @property({ serializable: false, visible: false })
+  // @property({ serializable: false, visible: false })
```

Comment:
The data doesn't seem to need to be processed by creator, so he doesn't have to mark it. Once marked, creator will try to identify his type. This leads to some unexpected errors.